src/cs/building/PowerPlant.cs

```diff
@@ -314,7 +318,10 @@ public partial class PowerPlant : Node2D {
 
 		// Plant no longer pollutes when it's powered off
 		Pollution = 0;
-
+		
+		// Changes the plant's color
+		this.Modulate = new Color(0.7f, 0.7f, 0.7f);
```

Comment:
Maybe you can give this color a name like "GRAY" rather that a hardcoded value. It would make this easier to read and less error prone in the case where that color will be reused.